src/aleph/config.py

```diff
@@ -7,6 +7,7 @@ def get_defaults():
     return {
         "logging": {
             "level": logging.WARNING,
+            "max_log_file_size": 1 * 1024 * 1024 * 1024,
```

Comment:
Would it not be more clear like this ?
```suggestion
            "max_log_file_size": 1_000_000_000,  # 1 GB
```

Reply:
Yes, I applied your suggestion.